### ISSUE TITLE
Replace setTimeout for polling in PhishingController

### ISSUE THREAD

**Issue by NiranjanaBinoy** [MetaMask/metamask-extension#15976]
Replace setTimeout for polling in updatePhishingLists() in PhishingController in a time window of 60 minutes.

**Comment by NiranjanaBinoy** [comment]
The corresponding PR has to be created in the controller repo.

Since the chrome alarms have to be used only for the extension, we would have to configure them at the API level to differentiate extension using mv3 from mobile and extension using mv2.
Can we introduce a new optional state variable with true value to identify extension with mv3 and use chrome alarms in that case?

Something similar to https://github.com/MetaMask/controllers/pull/861

**Comment by NiranjanaBinoy** [comment]
Closing this PR as the PhishingController does not require polling anymore. 
Quoting Mark's comment on the same from the channel

> [Mark Stacey](https://consensys.slack.com/archives/C01V1L10W2E/p1664226509781269?thread_ts=1664215606.626099&cid=C01V1L10W2E)
In some of these cases we can remove the polling altogether. We just did that for the phishing controller, for example. That can be removed from the list, as now it lazily fetches the config when it detects that it has been too long since it last checked.
